app/views/articles/_v2_form.html.erb

```diff
@@ -51,6 +49,12 @@
   </div>
 </div>
 
+<% cache("article-edit-scripts", expires_in: 8.hours) do %>
```

Comment:
I'm not sure we need to cache this right, there's nothing complicated to compute here, it's just the script content. 

If we decide to keep the cache, could we use `cache(cache_key_heroku_slug("article-v2-form-scripts"), expires_in: 8.hours)` here, this way it expires at each deploy

Reply:
Good catch!

Well, the only expense of it is `.html_safe`, it seems. I've looked into it, and  `show.html.erb` caching was there from the start, but none of the tags currently use any computing.
https://github.com/thepracticaldev/dev.to/blame/583160532c694a14a20bf8e99ac1546258181352/app/views/articles/show.html.erb#L232

I'm going to remove the caching here then.